manager.go

```diff
@@ -2083,3 +2021,65 @@ func cleanupUprobeEvents(pattern *regexp.Regexp, pidMask map[int]procMask) error
 	}
 	return cleanUpErrors
 }
+
+func (m *Manager) GetNetlinkSocket(nsHandle uint64, nsID uint32) (*NetlinkSocket, error) {
+	return m.netlinkSocketCache.GetNetlinkSocket(nsHandle, nsID)
+}
+
+type NetlinkSocketCache struct {
```

Comment:
Does this type need to be exported? I don't think the consumers of this library need to use it directly, correct?

Reply:
You are correct, oversight on my part, will push a PR on Monday fixing this

Reply:
👍